package.json

```diff
@@ -39,45 +39,46 @@
   "license": "MIT",
   "main": "./lib/index",
   "devDependencies": {
-    "autoprefixer": "^6.7.3",
+    "autoprefixer": "^7.2.3",
     "babel-cli": "^6.5.1",
     "babel-core": "^6.5.1",
-    "babel-eslint": "^6.1.2",
-    "babel-loader": "^6.2.4",
+    "babel-eslint": "^8.1.1",
+    "babel-loader": "^7.1.2",
     "babel-plugin-transform-es2015-spread": "^6.22.0",
     "babel-plugin-transform-es3-member-expression-literals": "^6.8.0",
     "babel-plugin-transform-es3-property-literals": "^6.8.0",
     "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.6.0",
+    "babel-preset-env": "^1.6.1",
     "babel-preset-stage-2": "^6.24.1",
-    "caniuse-support": "^0.4.0",
-    "cross-env": "^3.1.3",
-    "es5-shim": "^4.5.9",
-    "eslint": "^2.8.0",
-    "eslint-config-airbnb": "^8.0.0",
-    "eslint-config-jss": "^1.1.0",
-    "eslint-plugin-import": "^1.12.0",
-    "eslint-plugin-jsx-a11y": "^1.5.5",
-    "eslint-plugin-react": "^5.0.1",
+    "caniuse-db": "^1.0.30000784",
```

Comment:
why is it a direct dependency here?

Reply:
Because it should be always last version with fresh data and not depend on other packages (which can be use old version of caniuse-db)

Reply:
We can have control over the other package using it and keep it up to date there. If you put it in here, we have to keep it up-to-date in both packages. This is a hack, we shouldn't use if not necesserily

Reply:
But what if they do not update this package to the latest one? I can't trust to others packages, and don't want too:)

Reply:
who is they? it is you or me, otherwise you and me are gonna update it here and there. Don't create more jobs than we need.